app/src/main/java/guide/graphql/toc/ui/chapters/ChaptersFragment.kt

```diff
@@ -51,7 +52,11 @@ class ChaptersFragment : Fragment() {
                     ChaptersFragmentDirections.viewSections(
                         chapterId = chapter.id,
                         chapterNumber = chapter.number?.toInt() ?: -1,
-                        chapterTitle = chapter.title
+                        chapterTitle = if (chapter.number == null) chapter.title else getString(
```

Comment:
Instead of passing in the chapter title, we generate it here before passing it to the second fragment. This means we can use this attribute in the nav editor.